src/librustdoc/html/render/context.rs

```diff
@@ -43,6 +43,7 @@ use crate::try_err;
 /// It is intended that this context is a lightweight object which can be fairly
 /// easily cloned because it is cloned per work-job (about once per item in the
 /// rustdoc tree).
+#[derive(Clone)]
```

Comment:
This must be removed before this PR is merged; `Context` intentionally does not implement Clone (cf. `make_child_renderer()`). We should probably add something to its docs about this.

Reply:
Alright, shall work on it.

Reply:
Please feel free to ask for help here or on Zulip :)

Reply:
An easy to check that a type doesn't implement a trait is to have a doc code like this:

``````rust
/// ```compile_fail
/// let c = Context { ... };
/// c.clone();
/// ```
``````

:)

Reply:
Hey, are you suggesting that has been implemented or should be implemented?

Anyway - I'm almost finished removing the clones!

Reply:
Yea I take that back, this is proving to be a challenge haha